cfgmgr/portmgr.cpp

```diff
@@ -76,6 +77,47 @@ bool PortMgr::setPortAdminStatus(const string &alias, const bool up)
     return true;
 }
 
+bool PortMgr::setPortDHCPMitigationRate(const string &alias, const string &dhcp_rate_limit)
+{
+    stringstream cmd;
+    string res, cmd_str;
+    int ret;
+    int byte_rate = stoi(dhcp_rate_limit) * 406;
+
+    if (dhcp_rate_limit != "0")
+    {
+        // tc qdisc add dev <port_name> handle ffff: ingress
```

Comment:
Please use /* */ for multi-line comments

Reply:
fixed